nx-X11/lib/X11/XlibInt.c

```diff
@@ -2766,7 +2766,7 @@ void _XEatData(
    lib. This workaround had been implemented temporarily in a couple
    of X libs, see e.g. https://lists.x.org/archives/xorg-devel/2013-July/036763.html.
 */
-#include <X11/Xmd.h>  /* for LONG64 on 64-bit platforms */
+#include <nx-X11/Xmd.h>  /* for LONG64 on 64-bit platforms */
```

Comment:
Yeah, but I'll leave it for now. I have acted like that fix was not included yet. We should keep this clean until we fix it with one great commit.